docs/fundamentals/executor/executor-api.md

```diff
@@ -292,6 +292,35 @@ print(f'Text: {da[0].text}')
 ```text
 Text: hello world
 ```
+````
+
+
+
+````{tab} From local Python
+You can pull an `Executor` from the Jina Hub and use it directly as a Python object.
+
+
+````{admonition} From Jina Hub 
+:class: information
+The [hub](https://hub.jina.ai/) is our marketplace for `Executor`s.
+````
```

Comment:
Can the info box go above both tabs? I don't think it should be inside the local python tab :)